typescript/cli/src/tests/warp-apply.e2e-test.ts

```diff
@@ -35,7 +34,7 @@ const WARP_CORE_CONFIG_PATH_2 = `${REGISTRY_PATH}/deployments/warp_routes/ETH/an
 
 describe('WarpApply e2e tests', async function () {
   let chain2Addresses: ChainAddresses = {};
-  this.timeout(0); // No limit timeout since these tests can take a while
+  this.timeout(60000); // Long timeout since these tests can take a while
```

Comment:
I would have expected even longer than one minute actually
Btw, you can use underscores in numbers these days for legibility so `60000` -> `60_000`

Reply:
Fixed